client/app/account/user/credentials/delete/user-credentials-delete.html

```diff
@@ -0,0 +1,15 @@
+<div data-ng-controller="UserAccount.controllers.credentials.delete as $ctrl">
+
+    <div data-wizard
+         data-wizard-on-cancel="resetAction"
+         data-wizard-on-finish="deleteCredential"
+         data-wizard-title="i18n.user_credentials_delete_modal_title">
+
+        <div data-wizard-step>
+
+            <p data-ng-bind-html="tr('user_credentials_delete_modal_step1_question', [$ctrl.credential.name])"></p>
```

Comment:
tr? Should be $translate now :)

Reply:
Yes I need to rebase on `develop` branch.